streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -732,15 +732,16 @@ Task taskForInputPartition(final TopicPartition partition) {
         return tasks.values().stream().filter(t -> !t.isActive());
     }
 
+    // For testing only.
+    int commitAll() {
+        return commit(tasks.values());
+    }
+
```

Comment:
FWIW, I'd rather just inline it and avoid maintaining APIs that are just for tests in our production code.

Reply:
I see your point and thought about it, but I feel a little bit reluctant to remove it for the reason that it is indeed a handy call to make for testing purpose :)